### ISSUE TITLE
VS2022 Arm64 projects missing or do not load

### ISSUE THREAD

**Issue by aleks-f** [pocoproject/poco#3611]
@hmartinez82 when I regenerate Util with progen, the projects in the do not load properly in VS2022:
```
C:\Users\alex\source\repos\pocoproject\poco\Util\testsuite\TestSuite_vs170.vcxproj : 
error  : Cannot load project: project does not contain requested configuration debug_shared|ARM64. 
Please verify that .sln file is valid and uses only existing project configurations.
```
Foundation also does not load properly:
```
C:\Users\alex\source\repos\pocoproject\poco\Foundation\testsuite\TestApp_vs170.vcxproj : warning  : Platform 'ARM64' referenced in the project file 'TestApp' cannot be found.

error: Designtime build failed for project 'C:\Users\alex\source\repos\pocoproject\poco\Foundation\testsuite\TestLibrary_vs170.vcxproj', configuration 'debug_shared|ARM64'. IntelliSense might be unavailable.
	Set environment variable TRACEDESIGNTIME=true and restart Visual Studio to investigate.
C:\Users\alex\source\repos\pocoproject\poco\Foundation\testsuite\TestLibrary_vs170.vcxproj : warning  : The build tools for Visual Studio 2022 (v143) cannot be found. Install Visual Studio 2022 (v143) to build using the Visual Studio 2022 (v143) build tools.

C:\Users\alex\source\repos\pocoproject\poco\Foundation\testsuite\TestLibrary_vs170.vcxproj : warning  : Platform 'ARM64' referenced in the project file 'TestLibrary' cannot be found.
```
Other projects seem to have no arm64 configuration.

What is the status of VS2020 arm build?

**Comment by hmartinez82** [comment]
What was the ProGen line(s) used and what was the buildwin.cmd line used?

The current status for the ARM64 build is that ProGen is already working for the ProGen supported projects. I was able to build and test Foundation, CppUnit, XML, JSON, Encodings and Util already by running TestSuite for these projects in my Surface Pro X.

I called ProGen with /platform=Win32,x64,ARM64

One more thing. I had to `set VS160COMNTOOLS=%VS170COMNTOOLS%` before running ProGen because today it's hardcoded to use VS2019 and I only have VS2022
